library/components/TrendArrow.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="trendArrow">
+        <arrow-up-icon v-if="difference > 0" :size="iconSize" />
+        <minus-icon v-else-if="difference === 0" :size="iconSize" />
+        <arrow-down-icon v-else :size="iconSize" />
+    </div>
+</template>
+
+<script>
+export default {
+  name: "TrendArrow",
+  props: {
+    firstNumber: {
+      type: Number,
+      required: true,
+    },
+    secondNumber: {
+      type: Number,
+      required: true,
+    },
```

Comment:
current and past seems descriptive 🤷‍♀️

Reply:
Agreed, and I _hate_ "firstNumber" and "secondNumber", but "current" and "past" would be confusing if you wanted to use it to show something like "did we get more, the same, or fewer story points fixed this sprint than expected?" On that note "Trend" is probably not the right name either, I just can't come up with anything better. Shake fist @ brain.

Reply:
I actually have another suggestion, but I'll finish writing it. You can choose